cmd/nerdctl/volume_remove_linux_test.go

```diff
@@ -39,8 +40,8 @@ func TestVolumeRemove(t *testing.T) {
 	base := testutil.NewBase(t)
 
 	inUse := errdefs.ErrFailedPrecondition.Error()
-	malformed := errdefs.ErrInvalidArgument.Error()
-	notFound := errdefs.ErrNotFound.Error()
+	malformed := store.ErrInvalidArgument.Error()
+	notFound := store.ErrNotFound.Error()
```

Comment:
?

Reply:
Yeah, been back and forth on this.
I still think we have a general problem in the codebase with regard to errors management.
Since we generally do `fmt.Errorf` instead of using exported errors (`var ErrFoo = errors.New`), consuming code cannot test with `errors.Is`, making it hard to specialize behavior dependent on the type of error we get.

Some of my recent PR _do_ introduce better, finer grain and explicit error-ing.
Obviously, some of the errors should be shared across different sections of code, maybe into an `errs` subpackage.

Which brings the question of whether we want to try and reuse some (or all) of `containerd/errdefs` (for example `ErrNotFound`).

I have mixed feelings about whether we should do that or not, and I think I am leaning towards having our own independent `errs` package, as it will make things clearer that what nerdctl returns is nerdctl errors (even if down there, they happen to alias containerd errdef).

Thoughts?

Reply:
I'm not sure about this. Easier to merge this PR if this part can be omitted.

Reply:
Ok. Omitted with latest push.